README.md

```diff
@@ -180,53 +180,13 @@ service up`, regardless of which launch type is configured for your cluster (see
-                                     WARNING: Leaving this option blank can result in failure to launch container instances if an unavailable zone is chosen at random.
-   --security-group value            [Optional] Specifies a comma-separated list of existing security groups to associate with your container instances.
-                                     If you do not specify a security group here, then a new one is created.
-   --cidr value                      [Optional] Specifies a CIDR/IP range for the security group to use for container instances in your cluster.
-                                     This parameter is ignored if an existing security group is specified with the --security-group option. Defaults to 0.0.0.0/0.
-   --port value                      [Optional] Specifies a port to open on the security group to use for container instances in your cluster.
-                                     This parameter is ignored if an existing security group is specified with the --security-group option. Defaults to port 80.
-   --subnets value                   [Optional] Specifies a comma-separated list of existing VPC Subnet IDs in which to launch your container instances.
-                                     This option is required if you specify a VPC with the --vpc option.
-   --vpc value                       [Optional] Specifies the ID of an existing VPC in which to launch your container instances.
-                                     If you specify a VPC ID, you must specify a list of existing subnets in that VPC with the --subnets option.
-                                     If you do not specify a VPC ID, a new VPC is created with two subnets.
-   --force, -f                       [Optional] Forces the recreation of any existing resources that match your current configuration.
-                                     This option is useful for cleaning up stale resources from previous failed attempts.
-   --launch-type value               [Optional] Specifies the launch type. Options: EC2 or FARGATE. Overrides the default launch type stored in your cluster configuration.
-                                     Defaults to EC2 if a cluster configuration is not used.
-   --region value, -r value          [Optional] Specifies the AWS region to use. Defaults to the region configured using the configure command
-   --cluster-config value            [Optional] Specifies the name of the ECS cluster configuration to use. Defaults to the default cluster configuration.
-   --ecs-profile value               [Optional] Specifies the name of the ECS profile configuration to use. Defaults to the default profile configuration. [$ECS_PROFILE]
-   --aws-profile value               [Optional]  Use the AWS credentials from an existing named profile in ~/.aws/credentials. [$AWS_PROFILE]
-   --cluster value, -c value         [Optional] Specifies the ECS cluster name to use. Defaults to the cluster configured using the configure command
+ecs-cli up
```

Comment:
should this be wrapped with in-line code ticks, a la `ecs-cli up` ?

Reply:
Having the three tickets before and after make it a code block. You can view the rendered version [here] (https://github.com/SoManyHs/amazon-ecs-cli/blob/c33dc458cb4a3e3913b628f7c06577751b869b7d/README.md#creating-an-ecs-cluster)

Reply:
yeah totally missed those! ignore me